extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfigBuilder.java

```diff
@@ -1239,4 +1259,99 @@ public Token getToken() {
     public Logout getLogout() {
         return logout;
     }
+
```

Comment:
It seems to belong the utility code, some properties here can not be even supported by this builder but by various sub-builders

Reply:
I hope I am not confusing you, but let me put it this way: with this PR we control all the entry points and if user changes `OidcTenantConfig` instance after they passed it to Quarkus OIDC code, it has no effect. Which means after we have immutable copy where we control these 2 changeable fields (user info and tenant enabled), we can only rely on methods and state of former config class fields will not be changed (because Quarkus OIDC won't do it and noone else can).

Reply:
@michalvavrik 

>with this PR we control all the entry points and if user changes OidcTenantConfig instance after they passed it to Quarkus OIDC code, it has no effect.

The fact the test provider did it does not mean anyone would actually want to do it in the real application, why would they. And if they do, may be should fail instead, or log a message that the change happened and attempt to adapt... But it can bring so much complexity, without an obvious benefit, that I'm doubting we should do it